test/org/sagebionetworks/bridge/services/ParticipantServiceTest.java

```diff
@@ -626,6 +686,127 @@ public void notSettingStatusDoesntClearStatus() {
         verify(account, never()).setStatus(any());
     }
 
+    @Test
+    public void updateParticipantEmailCannotBeChanged() {
+        STUDY.setEmailVerificationEnabled(true);
+        mockHealthCodeAndAccountRetrieval("persisted@email.com", null);
+        when(account.getEmailVerified()).thenReturn(Boolean.TRUE);
+        
+        StudyParticipant participant = new StudyParticipant.Builder().copyOf(PARTICIPANT)
+                .withEmail("updated@email.com").withEmailVerified(Boolean.FALSE).build();
+        
+        participantService.updateParticipant(STUDY, ImmutableSet.of(), participant);
+        
+        verify(accountDao).updateAccount(accountCaptor.capture());
+        Account account = accountCaptor.getValue();
+        verify(account, never()).setEmail(anyString());
+        verify(account, never()).setEmailVerified(anyBoolean());
+        verify(accountWorkflowService, never()).sendEmailVerificationToken(eq(STUDY), eq(ID), anyString());
+    }
+
+    @Test
+    public void updateParticipantEmailCanBeAddedWithVerification() {
+        STUDY.setEmailVerificationEnabled(true);
+        mockHealthCodeAndAccountRetrieval(null, PHONE);
+        
+        // Set emailVerified = true to test that this is always set to false, regardless of submitted object
+        StudyParticipant participant = new StudyParticipant.Builder().copyOf(PARTICIPANT).withEmail("updated@email.com")
+                .withEmailVerified(Boolean.TRUE).withEmailVerified(Boolean.FALSE).build();
```

Comment:
This doesn't test what you think it tests. You call .withEmailVerified(true) and then immediately call .withEmailVerified(false), so the participant you're submitting has emailVerified = false.